#! /bin/sh -e

# DP: Build modules for ppp 2.4.3 and 2.4.4b1

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        #cd ${dir}gcc && autoconf
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        #rm ${dir}gcc/configure
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

--- pppdcapiplugin/Makefile~	2004-02-14 16:02:46.000000000 +0100
+++ pppdcapiplugin/Makefile	2005-02-27 16:35:32.922980771 +0100
@@ -18,7 +18,7 @@
 PPPSRCDIRS=/src/isdn/pppd
 
 ifeq ($(PPPVERSIONS),)
-PPPVERSIONS = 2.3.11 2.4.0 2.4.1 2.4.1b1 2.4.1b2 2.4.2b3 2.4.2 2.4.3
+PPPVERSIONS = 2.4.2 2.4.3 2.4.4b1
 endif
 
 PEERDIR=${DESTDIR}/etc/ppp/peers/isdn
